utils/CreateDataset.py

```diff
@@ -81,7 +86,7 @@ def __getitem__(self, index):
             metadata = None
             if meta_idx != -1:
                 metadata = self.metadata[meta_idx]
-        sample = {"sat_img": sat_img, "map_img": map_img, "metadata": metadata}
+        sample = {"sat_img": sat_img, "map_img": map_img, "metadata": metadata, "hdf5_path": self.hdf5_path}
```

Comment:
temporary patch before we use stac items. When using dataloaders, we can trace back path of hdf5 files.